packages/utils/src/lib/reports/md/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const NEW_LINE = '\n';
+export const TAB = ' ';
+export const SPACE = ' ';
```

Comment:
I have a radical suggestion - let's ditch these constants entirely :grin: It seems to me like a completely unnecessary abstraction.

Reply:
Yes! Next PR. 

There I can place that i feel ´NEW_LINE´ is quite helpful and `SPACE` is a visual anchor for me to not miss it by accident.